app/static/main.css

```diff
@@ -110,6 +110,26 @@ h1{
     margin-top: 30px; */
   }
 
+  .three-div-content .leftcontent {
+    /* can add structure styling here */
+  }
+
+  .three-div-content .middlecontent {
+    /* can add structure styling here */
+  }
+
+  .three-div-content .rightcontent {
+    /* can add structure styling here */
+  }
+
+  .two-div-content .leftcontent {
+    /* can add structure styling here */
+  }
+
+  .two-div-content .rightcontent {
+    /* can add structure styling here */
+  }
+
```

Comment:
Just placeholder for potential additional CSS on top of the bootstrap styling within the html docs.